libnetwork/drivers/bridge/setup_ip_tables_linux.go

```diff
@@ -314,6 +314,19 @@ func programChainRule(rule iptRule, ruleDescr string, insert bool) error {
 	return nil
 }
 
+func appendOrDelChainRule(rule iptRule, ruleDescr string, append bool) error {
+	operation := "disable"
+	fn := rule.Delete
```

Comment:
As we already have defined constants for these operations; should we just pass `rule.Delete` or `rule.Append` as argument?

There's various parts in libnetwork that pass boolean args to control flow, and it's been quite confusing.


(if there's more options beyond `Delete` and `Append` that are not supported in this function, we can return an error)

Reply:
OH! Nevermind; I see the `enable` variable is one that already existed, so we .. kind of must continue using booleans here.

Probably something to look at in a future cleanup, not for this PR.

Reply:
Yes - I set off in that direction, but decided a minimal fix was best for the moment.